pages/_app.tsx

```diff
@@ -17,6 +17,8 @@ const poppins = Poppins({
 });
 
 export default function App({ Component, pageProps }: AppProps) {
+  i18n.changeLanguage(pageProps.locale ?? "en");
```

Comment:
Is this necessary? My intuition tells me that the locale should automatically change because it's set in the document, but I might be totally wrong :)

Reply:
`i18next` doesn't look at the `lang` attribute to determine the language that needs to be used. So this is necessary, yea. Doing this in the `App` component feels really dirty, but well 😅